### PR TITLE
add j9vmClassUnloading into openj9

### PR DESCRIPTION
**Pull request by OscarQQ**
-related: backlog issue 387
[ci skip]

Signed-off-by: OscarQQ <Yixin.Qian@ibm.com>

### CODE REVIEW

**Comment by OscarQQ** [comment]
| Build       | Platform           | Java Version  | JDK Impl |
|:-------------:|:-------------:|:-----:|:-----------:|
| Grinder/10680     | ppc64le_linux | 8 | IBM|
| Grinder/10681    | s390x_linux      |   8 | IBM|
| Grinder/10682 | x86-64_linux      |    8 | IBM|
| Grinder/10683 | x86-64_windows      |    8 | IBM|
| not available(machines all offline) | s390x_zos     |    8 | IBM|
| Grinder/10685 | ppc64_aix     |    8 | IBM|
| Grinder/10706 | ppc64le_linux , s390x_linux ,x86-64_linux,x86-64_windows,x86-64_mac, ppc64_aix     |    8,11 | OpenJ9|

**Comment by OscarQQ** [comment]
@llxia

File: test/functional/VM_Test/build.xml [hunk 1]
```diff
@@ -155,6 +177,13 @@
 			<fileset dir="${data}" includes="ParentlessChild.class" />
 			<fileset dir="${properties}"/>
 		</jar>
+		<jar jarfile="${DEST}/classestoload.jar" filesonly="true">
```
[the remark this answers is not in the document]
Comment (llxia) [reply test/functional/VM_Test/build.xml:180]: Assuming it is used by `-Dj9vm.testing.tests.path=/home/jenkins/workspace/Grinder/openjdk-tests/TKG/../../jvmtest/functional/VM_Test`

File: test/functional/VM_Test/playlist.xml [hunk 2]
```diff
@@ -174,4 +174,30 @@
...
+		<variations>
+			<variation>Mode110</variation>
+			<variation>Mode301</variation>
+			<variation>Mode351</variation>
+			<variation>Mode501</variation>
+			<variation>Mode551</variation>
+			<variation>Mode610</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) -Xdump -Xint -Dj9vm.testing.tests.path=$(TEST_RESROOT) \
+	-cp $(Q)$(TEST_RESROOT)$(D)VM_Test.jar$(P)$(JVM_TEST_ROOT)$(D)VM$(D)lib$(D)asm-all.jar$(Q)  \
```
Comment (llxia) [inline test/functional/VM_Test/playlist.xml:188]: Please use `$(LIB_DIR)$(D)asm-all.jar`

File: test/functional/VM_Test/playlist.xml [hunk 3]
```diff
@@ -174,4 +174,30 @@
...
+	j9vm.runner.Menu -exe=$(SQ)$(TEST_JDK_HOME)$(D)bin$(D)java $(JVM_OPTIONS) -Xdump$(SQ) -jar=$(Q)$(TEST_RESROOT)$(D)VM_Test.jar$(Q) -test=$(Q)j9vm.test.classunloading.testcases$(Q) \
```
Comment (llxia) [inline test/functional/VM_Test/playlist.xml:189]: $(TEST_JDK_HOME)$(D)bin$(D)java -> $(JAVA_COMMAND)

File: test/functional/VM_Test/build.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (OscarQQ) [reply test/functional/VM_Test/build.xml:180]: Yes, it’s used by this

File: test/functional/VM_Test/playlist.xml (hunk 3 above)
Comment (OscarQQ) [reply test/functional/VM_Test/playlist.xml:189]: $(JAVA_COMMAND) has its own quotation, which is not expected

File: test/functional/VM_Test/build.xml [hunk 4]
```diff
@@ -170,6 +199,7 @@
 			<fileset dir="${data}" includes="**/*.class" />
 		</delete>
 		<delete dir="${databin}" />
+        <delete dir="${classUnloadingBin}" />
```
Comment (llxia) [inline test/functional/VM_Test/build.xml:202]: Please fix the indentation.

**Comment by llxia** [comment]
jenkins test extended all jdk8,jdk11

**Comment by OscarQQ** [comment]
[TestAttachErrorHandling_0](https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_ppc64_aix_Personal_testList_1/14/console) on ppc64_aix failed is related to https://github.com/eclipse/openj9/issues/3081
[cmdLineTester_EnableAssertionStatusTest_0](https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_x86-64_mac_Personal_testList_0/13/console) on x86-64_mac failed, needs investigation
these failures are not caused by j9vmClassUnloading

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**